src/global.scss

```diff
@@ -42,6 +42,30 @@ app-header{
  color: #8EC3C3;
 }
 
+html {
+  width: 100vw;
+  height: 100vh;
+    background-image: linear-gradient(to bottom, #8EC3C3 50%, #fff 50%);
+  background-color: #c38ea7;
+  margin-top: 0px !important; /* Status bar height on iOS 10 */
+
+  // margin-top: 20px; /* Status bar height on iOS 10 */
+  // margin-top: constant(safe-area-inset-top); /* Status bar height on iOS 11.0 */
+  // margin-top: env(safe-area-inset-top); /* Status bar height on iOS 11+ */
+}
```

Comment:
Is there no need to set different status bot height for iOS 10, 11.0, 11+ now?
If so, remove the commented-out code